table/tables/partition.go

```diff
@@ -939,19 +950,34 @@ func (t *partitionedTable) PartitionExpr() (*PartitionExpr, error) {
 	return t.partitionExpr, nil
 }
 
-func (t *partitionedTable) GetPartitionColumnNames() []model.CIStr {
+func (t *partitionedTable) GetPartitionColumnIDs() []int64 {
```

Comment:
Do we need to add a UT  for this function?

Reply:
Since it is used by GetPartitionColumnNames, and that is included in one UT, I don't think it is needed. But if you like I can create such UT?